### DISCUSSION TITLE
Deploying views in production

### DISCUSSION THREAD

**Discussion by jvelo** [planetscale/discussion#131] [FAQs] [answered]
Hello,

I've tried deploying a view from a dev branch to main, created via a `create or replace view ...` statement, though it seems the deploy request does not pick up on it, and states there's been no changes to the schema. Is this the expected behaviour ? Are there any other way to deploy a view to production ? 

Thanks in advance 🙇‍♂️

**Comment by jho14** [comment, marked as answer]
Hey @jvelo did you ever get this solved? About to go through the same process of deploying a view

**Reply by jgreet** [reply]
~~This should just work now. Support for creating, dropping, and altering views via deploy requests was added in April.~~

Edit: Rolled back due to some problems with nested views. Full support should be coming soon.

**Reply by jgreet** [reply]
Full support for deploying views, including nested views, to production branches is live now.

**Comment by jho14** [comment]
Hey @jgreet, thanks for the response. 

I seem to still be having this issue though. My views don't show up in the schema tab, however, the tables exist and can be queried through the CLI. Upon deploying to my development branch, I can't seem to find those tables when running ' show tables; '

It seems that my problem is that DRs are based on schema diffs and the views don't seem to be represented as a schema.

Am I missing something here?

**Reply by jgreet** [reply]
Sorry, no, I missed something here! We rolled back support for views while we we work on hammering out some issues with nested views.

If you'd like, we can flag your organization in to view support before it's enabled for everyone again. Just let us know which organization you're working in.

Apologies for the mix up.

**Reply by brunocangs** [reply]
Hi there! Has there been any updates on support for views? Asking since I'm interested in deploying a solution that uses Views to execute some operations that are harder to do on Prisma while maintaining type safety. Thanks!

**Reply by jgreet** [reply]
Yes, support for views was re-enabled for everyone in June https://github.com/planetscale/discussion/discussions/131#discussioncomment-2872138.  Let us know if you're having trouble creating them.
